client/my-sites/site-settings/disconnect-site/disconnect-survey.jsx

```diff
@@ -0,0 +1,96 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { isFreeJetpackPlan } from 'lib/products-values';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class DisconnectSurvey extends Component {
+	state = {
+		reasonSelected: 'onlyNeedFree',
+		renderInitial: true,
+	};
+
+	renderFollowUp() {
+		// placeholder
+		return <Card className="disconnect-site__question">{ 'follow-up' }</Card>;
+	}
+
+	logSelection = option => {
+		this.setState( {
+			reasonSelected: option.value,
+			renderInitial: false,
+		} );
+	};
+
+	getOptions() {
+		const { translate, site } = this.props;
+
+		const options = [
+			{ value: 'tooHard', label: translate( 'It was too hard to configure Jetpack' ) },
+			{ value: 'didNotInclude', label: translate( 'This plan didn’t include what I needed' ) },
+		];
+
+		if ( ! isFreeJetpackPlan( site.plan ) ) {
+			options.push( { value: 'onlyNeedFree', label: translate( 'This plan is too expensive' ) } );
```

Comment:
This is more of a question about the text and metrics, not related to the implementation.

`onlyNeedFree` seems to mean "the free plan includes the features I need."
"This plan is too expensive" seems to fundamentally mean something else.